CMakeLists.txt

```diff
@@ -24,17 +24,19 @@ set(
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-# add -fPIC to all targets
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard to use")
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard to use")
```

Comment:
What's the reason for the whitespace change?

Reply:
My mistake; it's fixed now